models/classes/user/events/LtiTestTakerCreatedEvent.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+namespace oat\taoLti\models\classes\user\events;
+
+use oat\tao\model\webhooks\WebhookSerializableEventInterface;
+
+class LtiTestTakerCreatedEvent implements WebhookSerializableEventInterface
+{
+    /** @var string */
+    private $testTakerUri;
+
+    public function __construct(string $testTakerUri)
+    {
+        $this->testTakerUri = $testTakerUri;
+    }
+
+    public function getName()
+    {
+        return __CLASS__;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getWebhookEventName()
+    {
+        return 'lti-test-taker-created';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function serializeForWebhook()
+    {
+        return [
+            'testTakerUri' => $this->testTakerUri,
+            'unit' => 1
```

Comment:
we need to document this somewhere , or to create a comment for the next developer

Reply:
Indeed, but I do not know where to document this or if is already self-documented.

IMO what is in the payload of this event should remain, cause there are listeners depending on this. I could not see documentation of previous payload documented too.

What do you suggest?

Reply:
I think this is done because of remote API contract. Even if we dont have. Once we will have we will document that on the other side, imo

Reply:
We could create webhhook.md in `doc` folder?

Reply:
I agree with @siwane and we will document it on the Lambda manager, so clients need to be compliant in case they will integrate.